e2e/testcafe-devextreme/tests/editors/selectBox/common.ts

```diff
@@ -1,14 +1,67 @@
 import { Selector } from 'testcafe';
 import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
-import { testScreenshot } from '../../../helpers/themeUtils';
+import { isFluent, testScreenshot } from '../../../helpers/themeUtils';
+import {
+  appendElementTo, setStyleAttribute,
+} from '../../../helpers/domUtils';
 import url from '../../../helpers/getPageUrl';
 import SelectBox from 'devextreme-testcafe-models/selectBox';
 import { createWidget } from '../../../helpers/createWidget';
-import { appendElementTo, setStyleAttribute } from '../../../helpers/domUtils';
 
 fixture.disablePageReloads`SelectBox placeholder`
   .page(url(__dirname, '../../container.html'));
 
+test('Last Item from last group in dropdown list has margin-bottom = 4px in Fluent Theme', async (t) => {
```

Comment:
Let's have a test on the list, because this test is no guarantee that it works correctly for other ddEditors.